### PR TITLE
update informal words

### PR DESCRIPTION
**Pull request by reza1615**
I added new informal words regexs

### CODE REVIEW

**Comment by halfak** [comment]
Hi @reza1615!  Thanks for your pull request.  It looks like you're behind master.  Could you try rebasing and pushing again?  If you're not sure what I mean, let me know and I'll try to see if I can resolve the issues.

**Comment by reza1615** [comment]
Hi @halfak  now it should be ok

File: revscoring/languages/persian.py [hunk 1]
```diff
@@ -5,58 +5,76 @@
...
+        r"ه؟ ؟شعر|و ?شعر|مادر|خو?اه?ر|آبجیت|هم ?شیره|داد)",
+    r"ر[زذ]ل\b",r"[کك]+[یي]+ر+\s*م?(ی|خر|(ب|)خور|تو[ی ]|مو |دهن)",
+    r"گا[يئی]ید[میي]",r"گاهييد[نه]",r"بگامت\b",r"(پدر|ننه|مادر|بابا|تخم)\s*سگ",
+    r"بی ?(شعور|شرف|ناموس)[یي]", r"\پريودى\؟",r"مادرت گا",
+    r"تنت میخاره", r"به کیرم",r"به گا ميدم",r"\bبگای?د",
+    r"برای مادرت",r"دیو[سث]\b",r"\bننتو",r"گوزید[نه]?",r"\bگه نخور",r"انگشت به كون",
+    r"\bچاکت\b",r"\bجنده",r"گه اضاف[يی] خورد[هیي]",r"خاک تو سرت",
+    r"[کك][یي]رم\b",r"ر[یي]د[همی]\b",r"[کك]ون ?ده",r"[کك]س ?ده",
+    r"گا[یي]ش",r"ب[کك]ن ب[کك]ن",r"([کك]+[یي]+ر+)ی+\b",
+    r"(به پشت|دمر|دمرو) بخواب",r"خایه لیس",r"حسن کلیدساز",r"\bکره خر",
```
Comment (halfak) [inline revscoring/languages/persian.py]: It looks like there are two opening parens in the first regex on this line.  Am I reading that char mistakenly?

File: revscoring/languages/persian.py [hunk 2]
```diff
@@ -5,58 +5,76 @@
...
+    r"\b(ن?می‌? ?|ب|ن)([یا]رز|[یا]فت|[یا]فراز|[یا]فروز|[یا]ندیش|[یا]نگیز|ایست|"+
+        r"ی?[اآ]زار|ی?[اآ]فرین|ی?[اآ]مرز|ی?[اآ]میز|ی?[اآ]ور|ی?[اآ]ویز|بار|باز|"+
+        r"باف|بال|بخش|بر|بلع|بند|بوس|بین|پاش|پذیر|پر|پراکن|پرداز|پرس|پرست|پرور|"+
+        r"پز|پسند|پلک|پندار|پوس|پوش|پیچ|پیوند|تاب|تپ|تراش|ترس|ترش|ترک|جنب|جنگ|جه|"+
+        r"جو|جوش|چاپ|چپ|چر|چرب|چرخ|چسب|چش|چک|چین|خار|خر|خز|خشک|خند|خواب|خور|دار|"+
+        r"در|درخش|دزد|دم|دو|دوز|دوش|رس|رقص|رنج|ریز|رین|زار|زن|ساب|ساز|سپار|سنج|شاش|"+
+        r"شکاف|شکف|شکن|شمر|شناس|شنو|شور|طپ|طلب|غر|غلت|فرست|فروش|فریب|فهم|قاپ|کار|"+
+        r"کن|کوب|گذار|گذر|گرد|گری|گریز|گستر|گمار|گند|گوز|گیر|لرز|لغز|لنگ|لول|لیس|"+
+        r"ماس|مال|مک|میر|ناز|نال|نگر|نواز|نورد|نویس|هراس|ورز|وز|یاب)(ه|ن)\b",
+    r"\b(ن?می‌? ?|ب|ن)(پا|زا|کاه|گا)ن\b",
```
Comment (Ladsgroup) [inline revscoring/languages/persian.py]: unbalanced parentheses

File: revscoring/languages/persian.py [hunk 3]
```diff
@@ -5,58 +5,76 @@
...
+    #wikipedia words
+    r"\b(بحث\:|بحثم|کاربر\:|میان‌ویکی|ویکی‌? ?فا\b)\b"
 ]
 
 badwords = [
+    #Fingilish bad words 
     r"(madar|nanae|zan|khahar)\s*?(ghahbeh|ghahveh|ghabe|jendeh?|be khata)",
-    r"khar madar", r"khar kos deh",
+    r"khar madar",r"khar kos deh",
     r"([qkc]+o+s+|[qkc]+o+n+|[qkc]+i+r+|m+e+g+h+a+d)\s*?((va|o)\s*?[qkc]oon|"+
```
Comment (Ladsgroup) [inline revscoring/languages/persian.py]: another unbalanced parentheses

File: revscoring/languages/persian.py (hunk 1 above)
Comment (ebraminio) [inline revscoring/languages/persian.py]: Yep, that is just visually mirrored not logically (which matters) two open parens.

File: revscoring/languages/persian.py [hunk 4]
```diff
@@ -5,57 +5,78 @@
...
+    r"\b(ارادتمندم?|ببخشید|بفرما|جناب|خدمت[تم][وا]?ن?|خوبی|خودت[وا]ن|خودم[وا]ن|خوشحالم|" +
+    r"خوشم|درود|دوستدار|سپاسگزارم|سلام|شرمنده|شماست|فلانی|مبارک|متشکرم|محترمت[وا]ن|" +
+    r"مخالفم|مرسی|ممنونم?|منظورت[وا]ن|موافقم|نظر[تم][وا]?ن?|نظرم|یادت[وا]ن)\b",
+    # wikipedia words
+    r"\b(بحث\:|بحثم|کاربر\:|میان‌ویکی|ویکی‌? ?فا)\b"
 ]
 
 badwords = [
+    # Fingilish bad words
     r"(madar|nanae|zan|khahar)\s*?(ghahbeh|ghahveh|ghabe|jendeh?|be khata)",
```
Comment (halfak) [inline revscoring/languages/persian.py]: Regretfully, we can't support spaces in words right now.  See https://github.com/wiki-ai/revscoring/issues/164

If there's some way we can record these multi-word badwords for future use, but only include single-word badwords right now, that would be most desirable.

**Comment by Ladsgroup** [comment]
Needs rebase

**Comment by halfak** [comment]
Looks like this is covered by some work that @Ladsgroup did.  I'm going to close, but please re-open if I'm mistaken.

**Pull request closed without merging**